changelog/_unreleased/2022-05-11-fix-apps-break-extension-listing.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Fix apps breaking extension listing in administration
+author: Silvio Kennecke
+author_github: @silviokennecke
+---
+# Core
+* load app label and description from translations in `\Shopware\Core\Framework\Store\Services\ExtensionLoader::prepareAppData`
```

Comment:
```suggestion
* Changed `\Shopware\Core\Framework\Store\Services\ExtensionLoader::prepareAppData` to load app label and description from translations
```

Changelog entries have to start with certain keywords like:
* Added
* Changed
* Deprecated
* Removed